pkg/clusteragent/admission/controllers/webhook/controller_v1.go

```diff
@@ -242,3 +242,14 @@ func (c *ControllerV1) getWebhookSkeleton(nameSuffix, path string) admiv1.Mutati
 
 	return webhook
 }
+
+func (c *ControllerV1) getAdmiV1FailurePolicy() admiv1.FailurePolicyType {
+	switch c.config.getFailurePolicy() {
+	case "Ignore":
+		return admiv1.Ignore
+	case "Fail":
+		return admiv1.Fail
```

Comment:
we should support lower case config as well
```suggestion
	case "Ignore", "ignore":
		return admiv1.Ignore
	case "Fail", "fail":
		return admiv1.Fail
```

Reply:
Please add a test case for lower-case config 🙇

Reply:
I added the requested change, but I wonder, would it be better to simply lowercase the value and ignore case all together?

Reply:
it would cover more cases yes, thanks 👍